templates/web/zurich/admin/report_edit.html

```diff
@@ -131,22 +133,20 @@
 </script>
 </p>
 
-<p><span class="mock-label">[% loc('State:') %]</span> <select name="state"  id="state">
-    <option value="">--</option>
-  [% FOREACH s IN states %]
-    <option [% 'selected ' IF s.state == problem.state %] value="[% s.state %]">[% s.trans %]</option>
-  [% END %]
-</select></p>
-
-<p id="automatic-reply" class="report-edit-action">
-    <span class="mock-label">Automatische Antwort</span>
-    <input type="checkbox" name="send_rejected_email" id="send_rejected_email" value="1" />
+<p> 
+    <span class="mock-label">[% loc('State:') %]</span>
+    <select name="state"  id="state">
+        <option value="">--</option>
+    [% FOREACH s IN states %]
+        <option [% 'selected ' IF s.state == pstate %] value="[% s.state %]">[% s.trans %]</option>
+    [% END %]
+    </select>
 </p>
 
 <script type="text/javascript">
 $(function(){
 
-    $('.assignation select').change(function(){
+    $('.assignation__select').change(function(){
```

Comment:
Some of the logic in this and the `#state` change handler isn't quite right, but it's fixed in a later commit.